src/OpenSage.Game/Data/W3d/W3dVertexMapperArgs.cs

```diff
@@ -85,23 +86,23 @@ public static W3dVertexMapperArgs Parse(string value)
                 switch (mapperArgName)
                 {
                     case "UPerSec":
-                        result.UPerSec = float.Parse(mapperArgValue);
+                        result.UPerSec = ParseFloat(mapperArgValue);
                         break;
 
                     case "VPerSec":
-                        result.VPerSec = float.Parse(mapperArgValue);
+                        result.VPerSec = ParseFloat(mapperArgValue);
                         break;
 
                     case "UScale":
-                        float.TryParse(mapperArgValue, out result.UScale);
+                        TryParseFloat(mapperArgValue, out result.UScale);
```

Comment:
I was wondering - `float.TryParse` is used twice this way in this file, but the return value is never checked.

Is this because these values might be corrupted / invalid but it's not a critical issue, and in that case you want to set the out variable to 0?

Reply:
Yes, some of them contain invalid data (specifically, some have no value but are missing the comment character - `;` - before the comment).

Actually all of the cases could / should use `TryParseFloat`, but so far I only changed it for the ones I needed to for the actual data.

Reply:
I did some regex-fu and replaced all instances with `TryParseFloat`.